functions/src/compute-statistics.ts

```diff
@@ -1,15 +1,21 @@
 import * as functions from "firebase-functions";
 import * as firebase from "firebase-admin";
-import * as firebaseTools from "firebase-tools";
 
 import { Config } from "./config";
-import { updateStats } from "./update-stats";
+import { getStatsRefsToUpdate } from "./update-stats";
 import { Vote } from "./cast-vote";
 
 const config = functions.config() as Config;
 const computeStatisticsConfigs = config.features.compute_statistics;
 const db = firebase.firestore();
 
+//not the best but otherwise values show as NaN
+const defaultCounters = {
+  great: 0,
+  notThatGreat: 0,
+  notGreatAtAll: 0
+};
```

Comment:
Why do they show as NaN?

Reply:
I have no idea

Reply:
Take the time to understand your code :). Come back with the explanation.